db/migrate/20151220093801_remove_spaces_from_smart_variable_key.rb

```diff
@@ -0,0 +1,13 @@
+class RemoveSpacesFromSmartVariableKey < ActiveRecord::Migration
+  def up
+    VariableLookupKey.where("key LIKE '% %'").each do |lookup_key|
+      lookup_key.update_attribute(:key, lookup_key.key.tr(' ', '_'))
+    end
+  end
+
+  def down
+    VariableLookupKey.where("key LIKE '%\\_%'").each do |lookup_key|
+      lookup_key.update_attribute(:key, lookup_key.key.tr('_', ' '))
```

Comment:
There might be keys that already contain underscores, not sure this is a good idea as this will break them. I am not sure that we need a down migration at all.